app/views/search/_granule_list.html.erb

```diff
@@ -1,76 +1,68 @@
       <section id="granule-list" class="master-overlay-no-min granule-list" data-bind="visible: ui.collectionsList.focused()" tabIndex="2" data-master-overlay-title="Granules">
         <nav class="master-overlay-nav">
           <a class="master-overlay-back master-overlay-hide-secondary" href="#" data-bind="click: function() {ui.projectList.hideFilters(); ui.collectionsList.unfocusCollection(); }">
-            <i class="fa fa-arrow-circle-o-left"></i> Back to Collections
+            <i class="fa fa-chevron-circle-left"></i> Back to Collections
           </a>
         </nav>
         <!-- ko if: ui.collectionsList.focused() && ui.collectionsList.focused().granules()  -->
         <!-- ko with: ui.collectionsList.focused() -->
         <header class="with-options">
-          <h2 data-bind="text: collection.title"></h2>
-          <!-- ko with: collection -->
+          <div class="collection-title-filters">
+            <h2><span data-bind="text: collection.title"></span> <!-- ko with: collection --><a href="#" class="master-overlay-forward collection-title-link" data-master-overlay-forward="Granules" data-bind="click: $root.ui.collectionsList.showCollectionDetails" title="View collection details">View details</a><!-- /ko --></h2>
+            <div id="datasource-caveats"></div>
+            <div class="master-overlay-info">
+
+              <!--ko ifnot: $data.collection.cwic()-->
+                <p class="granule-sort">Sort by: <select id="granule-sort" data-bind="options: granules().query.sortOptions, optionsText: 'name', optionsValue: 'value', value: granules().query.sortKey"></select></p>
+
+                <p class="granule-id-search">
+                  Granule Search: <input id="granule-ids" type="text" placeholder="Search Single or Multiple Granule IDs..." data-bind="value: $data.collection.granuleIds, valueUpdate: 'keyup'" data-tooltip="true" title="Wildcards: * (asterisk) matches any number of characters, and ? (question mark) matches exactly one character.
+    Delimiters: Separate multiple granule IDs by space, comma, or new line."/>
+                </p>
+              <!--/ko-->
+
+              <!-- ko with: collection -->
+                <!-- ko if: has_granules -->
+                <a href="#" data-bind="visible: !$root.project.isSearchingGranules($data), click: $root.ui.projectList.showFilters, css: {'button-highlighted': granuleFiltersApplied()}" class="master-overlay-hide-parent master-overlay-show-secondary granule-filter-link" title="Filter granules">Granule filters</a>
+                <a href="#" data-bind="visible: $root.project.isSearchingGranules($data), click: $root.ui.projectList.hideFilters, css: {'button-highlighted': granuleFiltersApplied()}" class="master-overlay-hide-parent master-overlay-hide-secondary granule-filter-link" title="Hide granule filters">Granule filters</a>
```

Comment:
If we add the `fa-info-circle` to the view details link, then we should also add the filter icon on the Granule Filters link as well.